docs/BrowserObject.md

```diff
@@ -82,14 +83,51 @@ console.log(browser.config)
         connectionRetryTimeout: 120000,
         connectionRetryCount: 3,
         specs: [ 'err.js' ],
-        fakeUser: 'maxmustermann', // <-- custom option
-        fakePassword: 'foobar', // <-- custom option
+        fakeUser: 'maxmustermann', // <-- custom configuration
+        fakePassword: 'foobar', // <-- custom configuration
+        baseUrl: 'example.com', // <-- custom configuration
```

Comment:
`baseUrl` is already defined a bit more up and also is not a custom configuration.

Reply:
Ah good catch I missed that `baseUrl`, will remove the duplicate.

> is not a custom configuration

Well, is it not? I understand it's a "reserved" key in `browser.options`, so it's not "custom" in that sense. But it's still defined alongside other configurations in `wdio.conf.js` with a custom value. And to me, it's confusing when to reference `browser.options` and when to reference `browser.config` when the same key is defined in both.

Reply:
To be honest, I am not quite sure why we still have both. It is ambiguous. I will create an issue getting rid of `browser.config`.